HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -816,7 +816,7 @@ def self.process_load_walls(mj, bldg_design_loads, all_space_design_loads)
       end
       if not wall.attached_to_space_idref.nil?
         space_design_loads = all_space_design_loads[wall.attached_to_space_idref]
-        if wall.is_exterior
+        if wall.is_exposed
```

Comment:
@yzhou601 We are already only looping through thermal boundary surfaces, what additional surfaces are limited by using `is_exposed`? I can't help but feel that we don't need any if statement at all here. Same for foundation walls below.

Reply:
So the is_exterior was used to be `exterior_adjacent_to` to be outside (or ground for foundation wall) before. Manual J has a concept of whether the surface contacts outdoor air, which is what this is_exposed tries to handle it better. It's not all the thermal boundary surfaces, but not only exterior walls. For example, garage is assumed to be open so by default the garage wall should be counted as an exposed wall, and in the Cobb example, east walls are connecting to an open stairwell which is also considered exposed walls. Exposed walls are summed up to calculate the wall area ratio for space infiltration load calculations.